ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -8,9 +8,56 @@
 
 import Foundation
 
+public protocol MutexType: class {
```

Comment:
Can you make this protocol and the other added types also `private` or `internal`? There's no reason to expose them since they can't be used with `Atomic` externally.